client/app/host/HostNavbar.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+import SettingsMenu from '@/components/SettingsMenu';
+import { Menubar, MenubarMenu, MenubarTrigger } from '@/components/ui/menubar';
+import { FileDown } from 'lucide-react';
+
+const HostNavbar = () => {
+    const menuItems = [{ icon: <FileDown />, function: false }];
+
+    return (
+        <div className="flex w-full">
+            <Menubar>
+                <MenubarMenu>
+                    {menuItems.map((item, index) => (
+                        <MenubarTrigger key={index}>{item.icon}</MenubarTrigger>
+                    ))}
+                </MenubarMenu>
+            </Menubar>
```

Comment:
I think the `MenuBar` can go ahead and be removed.  Just replace it with a div that contains buttons.  Also remove the `menuItems` array here and just have the buttons in the HTML.